roles/edpm_pre_adoption_validation/molecule/adoption-positive/molecule.yml

```diff
@@ -4,7 +4,10 @@ dependency:
 driver:
   name: podman
 platforms:
-- name: edpm-0
+# NOTE(gibi): we are forcing FQDN as hostname here. This is not super realistic
+# but works for our testing purposes to simulate that hostname -f returns
+# FQDN
+- name: edpm-0.localdomain
```

Comment:
ya ok that works.
i have seen people set /etc/hostname to an fqdn even if they should not do that but
i think this is valid for testing.